apiary.apib

```diff
@@ -248,11 +248,75 @@ You can find a (very) minimal PHP code example, showing how start the OAuth 2 au
 
 A Teamleader PHP SDK is under development. In the meanwhile, we recommend using packages such [Guzzle](http://docs.guzzlephp.org/en/stable) or the [League OAuth2 client](http://oauth2-client.thephpleague.com).
 
-## Changelog
+## Changes & upgrades
 
-You can follow our API progress and ongoing proposals via our [API Blueprint GitHub repository](https://github.com/teamleadercrm/api).
+We want to keep this API backwards compatible as long as possible, but at the same time we need to make changes to it to match our evolving product.
+
+Therefor we assigned a dated version to your API client when it was created.
+This version controls the API and webhooks behaviour for you API client.
+When we change the API in a backwards-incompatible way, we release a new dated version.
+But to avoid breaking your code, you stay on your assigned version until you [upgrade your API version](./#upgrading-your-api-version).
```

Comment:
PP: sentence should not start with `but`. this indicates that it's a subsentence instead.